eng/packages/http-client-csharp/generator/Azure.Generator/test/TestHelpers/Configuration.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "output-folder": "./outputFolder",
   "project-folder": "./projectFolder",
-  "namespace": "sample.namespace",
+  "namespace": "sample.test",
```

Comment:
`namespace` is reserved in CSharp, it breaks the indentation of the generated code.